single_kernel_mongo/managers/mongodb_operator.py

```diff
@@ -707,6 +708,8 @@ def set_password(self, username: str, password: str | None = None) -> tuple[str,
         if user == MonitorUser:
             # Update and restart mongodb exporter.
             self.mongodb_exporter_config_manager.configure_and_restart()
+        if user == LogRotateUser:
```

Comment:
nit: could you add a comment for consistency like we do on the 2 previous conditions